src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,54 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+interface Props {
+  movie: Movie
+}
+
+export const MovieCard: React.FC<Props> = (props) => {
+  const { movie } = props;
+
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={movie.imgUrl}
```

Comment:
you can desctructure the 'movie' 
```suggestion
            src={imgUrl}
```